src/step-13-typescript/config.ts

```diff
@@ -0,0 +1,11 @@
+import { join } from 'path'
+import envSchema from 'env-schema'
+import S from 'fluent-json-schema'
+
+const schema = S.object()
+  .prop('JWT_SECRET', S.string().required())
```

Comment:
can we create a type for this configuration schema? so we can use it when we configure the server and the plugins?